src/libs/actions/Report.js

```diff
@@ -516,6 +516,26 @@ function openReport(reportID, participantLoginList = [], newReportObject = {}, p
         },
     ];
 
+    // If we're going to open a failed money request report, remove the report preview action in the failure data
+    const currentReport = ReportUtils.getReport(reportID);
+    const reportCreationErrors = _.chain(lodashGet(currentReport, 'errorFields.createChat')).compact().value();
```

Comment:
Oh yeah good point. I think I chained because I thought that was needed to convert the object into a list. But I don't think chain does that anyways and compact can act on an object.